modules/abstract-eth/src/lib/utils.ts

```diff
@@ -672,37 +676,98 @@ export function getV1WalletInitializationData(walletOwners: string[], salt: stri
 }
 
 /**
- * Returns the create address method calling data for v1 wallets
+ * Returns the create address method calling data for v1, v2, v4 forwarders
  *
  * @param {string} baseAddress - The address of the wallet contract
  * @param {string} salt - The salt for address initialization transactions
+ * @param {string} feeAddress - The fee address for the enterprise
  * @returns {string} - the createForwarder method encoded
  */
-export function getV1AddressInitializationData(baseAddress: string, salt: string): string {
+export function getV1AddressInitializationData(baseAddress: string, salt: string, feeAddress?: string): string {
```

Comment:
nit: the method name is misleading

Reply:
There are couple of instances where naming are misleading in sdk,  just because when they were added it was only for v1, but then later those methods were reused for other wallet/forwarder versions as well. I think we probably should update all of them in one PR as that will be a breaking change

Reply:
Have created a follow up ticket with all the method/variable names to be renamed. https://bitgoinc.atlassian.net/browse/WIN-1964